test/unit/org/apache/cassandra/db/commitlog/CommitLogTest.java

```diff
@@ -118,6 +119,8 @@ public static Collection<Object[]> generateData()
 
     public static void beforeClass() throws ConfigurationException
     {
+        KEYSPACE1 = "CommitLogTest" + ksIter;
+        KEYSPACE2 = "CommitLogTestNonDurable" + ksIter++;
```

Comment:
This {{beforeClass}} static method is run only once at the beginning of the suite, so the keyspace name suffix is always zero, even across different sets of parameters.

Reply:
That's what logic dictates yes. But when using the `RepeatableRunner` it does indeed fail hence why I went this route:
```
[junit-timeout] Testcase: org.apache.cassandra.db.commitlog.GroupCommitLogTest:	Caused an ERROR
[junit-timeout] Cannot add existing keyspace "CommitLogTest"
[junit-timeout] org.apache.cassandra.exceptions.AlreadyExistsException: Cannot add existing keyspace "CommitLogTest"
[junit-timeout] 	at org.apache.cassandra.schema.SchemaTestUtil.announceNewKeyspace(SchemaTestUtil.java:47)
[junit-timeout] 	at org.apache.cassandra.SchemaLoader.createKeyspace(SchemaLoader.java:267)
[junit-timeout] 	at org.apache.cassandra.db.commitlog.CommitLogTest.beforeClass(CommitLogTest.java:164)
[junit-timeout] 	at org.apache.cassandra.db.commitlog.GroupCommitLogTest.setCommitLogModeDetails(GroupCommitLogTest.java:47)
[junit-timeout] 	at org.apache.cassandra.RepeatableRunner.run(RepeatableRunner.java:84)
```

In any case this PR is wrong anyway